packages/dashboard/src/components/sidePanel/sections/propertiesAlarmSection/index.test.tsx

```diff
@@ -10,6 +9,18 @@ import { QueryWidget } from '../../../../customization/widgets/types';
 import { DashboardState } from '../../../../store/state';
 import { configureDashboardStore } from '../../../../store';
 
+import { useAssetDescriptionMapAsync } from '../../../../hooks/useAssetDescriptionMapAsync';
```

Comment:
is there a way we can do this that doesn't require mocking the hook?

Reply:
Yup good callout, refactored.